store/modules/publish.ts

```diff
@@ -200,6 +200,7 @@ export const actions: Actions<State, RootState> = {
     commit(types.UPDATE_STATUS, status);
   },
 
+  // TODO seems to be not in used
```

Comment:
red herring code path that confused me for a little bit. so marking it for future